packages/contracts-bedrock/src/dispute/lib/LibPosition.sol

```diff
@@ -156,7 +162,13 @@ library LibPosition {
         returns (Position ancestor_)
     {
         // This function only works for positions that are below the upper bound.
-        if (_position.depth() <= _upperBoundExclusive) revert ClaimAboveSplit();
+        if (_position.depth() <= _upperBoundExclusive) {
+            assembly {
+                // Revert with `ClaimAboveSplit()`
```

Comment:
Why not use the error type here?

Reply:
Because it requires us to import `Errors.sol`, which imports `LibUDT.sol`, which imports `LibPosition.sol`. Solc can't resolve the dependency, even though it's all kosher...